kybra/compiler/kybra_generate/src/generators/func.rs

```diff
@@ -0,0 +1,49 @@
+use proc_macro2::TokenStream;
```

Comment:
Please change try_from_vm_value to try_from_vm_value_impls

Reply:
Done with 1f4dcde16881249266f41c1f6cb87d3701a4181f